plugins/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -89,6 +89,7 @@
         description: Whether SSL certificates should be verified for HTTPS requests (deprecated). defaults to True.
         type: bool
         default: true
+        aliases: [ verify_ssl ]
```

Comment:
Why add this alias (especially without modifying the argument spec)?

Reply:
oops, wrong branch